README.md

```diff
@@ -0,0 +1,26 @@
+# Field Formatter
+
+[Field Formatter](https://www.drupal.org/project/field_formatter) module adds formatters for entity reference fields to output only a specific field or field value using a specific formatter w/o needing to configure media view mode and configuring it to display properly. This module provides two formatters: *Field formatter with inline settings* and *Field formatter from view display*.
+
+## Installation
+
+1. Download [Field Formatter](https://www.drupal.org/project/field_formatter) from [GitHub](https://github.com/drupal-media/field_formatter).
+2. Install it in the [usual way](https://www.drupal.org/documentation/install/modules-themes/modules-8).
+
+## Usage
+
+1. Create entity reference field
+  * On `admin/structure` choose **Content types**
+  * Choose content type on which you want to use entity reference field, for example: *Article*, and click **Manage fields** and then **+ Add field**
```

Comment:
This reads a bit strange. What about "Choose content type entity reference field should be added to." or something similar.